src/views/MapPCAETView.vue

```diff
@@ -0,0 +1,8 @@
+<script setup lang="ts">
+import { FooterArea } from 'cooperation_jn_common_ui'
+</script>
```

Comment:
Perhaps when the component is mounted set the current View to viewList['map-pcaet']